src/connectdlg.cpp

```diff
@@ -42,43 +42,51 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
+                             tr ( "Select a directory to list the servers registered with it." ) + "<br/>" +
+                             tr ( "You can add custom directories in Advanced Settings." );
+    lblDirectory->setWhatsThis ( strDirectoryWT );
+    lblDirectory->setToolTip ( strDirectoryWT );
     cbxDirectory->setWhatsThis ( strDirectoryWT );
+    cbxDirectory->setToolTip ( strDirectoryWT );
+
+    QString strDirectoryAN = tr ( "Directory combo box" );
+    lblDirectory->setAccessibleName ( strDirectoryAN );
     cbxDirectory->setAccessibleName ( strDirectoryAN );
 
     // filter
     QString strFilterWT = "<b>" + tr ( "Filter" ) + ":</b> " +
                           tr ( "Filters the server list by the given text. Note that the filter is case insensitive. "
                                "A single # character will filter for those servers with at least one person connected." );
-    QString strFilterAN = tr ( "Filter edit box" );
     lblFilter->setWhatsThis ( strFilterWT );
+    lblFilter->setToolTip ( strFilterWT );
     edtFilter->setWhatsThis ( strFilterWT );
+    edtFilter->setToolTip ( strFilterWT );
 
+    QString strFilterAN = tr ( "Filter edit box" );
     lblFilter->setAccessibleName ( strFilterAN );
     edtFilter->setAccessibleName ( strFilterAN );
 
     // show all mucisians
-    chbExpandAll->setWhatsThis ( "<b>" + tr ( "Show All Musicians" ) + ":</b> " +
-                                 tr ( "Uncheck to collapse the server list to show just the server details. "
-                                      "Check to show everyone on the servers." ) );
+    QString strExpandAllWT = "<b>" + tr ( "Show All Musicians" ) + ":</b> " +
+                             tr ( "Uncheck to collapse the server list to show just the server details. "
+                                  "Check to show everyone on the servers." );
+    chbExpandAll->setWhatsThis ( strExpandAllWT );
+    chbExpandAll->setToolTip ( strExpandAllWT );
 
     chbExpandAll->setAccessibleName ( tr ( "Show all musicians check box" ) );
 
     // server list view
-    lvwServers->setWhatsThis ( "<b>" + tr ( "Server List" ) + ":</b> " +
-                               tr ( "The Connection Setup window lists the available servers registered with "
-                                    "the selected directory. Use the Directory dropdown to change the directory, "
-                                    "find the server you want to join in the server list, click on it, and "
-                                    "then click the Connect button to connect. Alternatively, double click on "
-                                    "the server name to connect." ) +
-                               "<br>" + tr ( "Permanent servers (those that have been listed for longer than 48 hours) are shown in bold." ) +
-                               "<br>" + tr ( "You can add custom directories in Advanced Settings." ) );
+    QString strServersWT = "<b>" + tr ( "Server List" ) + ":</b> " +
```

Comment:
I'm a bit worried about the server list as it could be annoying to see a long text. Otherwise all is good.

Reply:
I'd suggest to shorten it drastically for the hover menu while leaving the what's this text as is.

Reply:
OK, shorter version done.